htdocs/product/class/product.class.php

```diff
@@ -2643,7 +2652,10 @@ public function load_stats_commande($socid = 0, $filtrestatut = '', $forVirtualS
                     $this->stats_commande['qty'] -= $adeduire;
                 }
             }
-
+            $hookmanager->initHooks(array('productdao'));
+            $parameters = array('socid' => $socid, 'filtrestatut' => $filtrestatut, 'forVirtualStock' => $forVirtualStock);
+            $reshook = $hookmanager->executeHooks('loadStatsCommande', $parameters, $this, $action);
```

Comment:
Maybe use English name for all new hooks?

Example for this hook :

`loadStatsCustomerOrder` instead `loadStatsCommande`

Reply:
i kept function name so i dont know, what do you prefer @eldy ?

Reply:
For any new function, method we must try to introduce english.